src/sidebar/app/actions.js

```diff
@@ -101,7 +101,8 @@ export function createNote(content = '', origin) {
     id,
     content,
     origin,
-    lastModified: new Date()
+    lastModified: new Date(),
```

Comment:
This is a bit tedious, we shoud use the server timestamp here. I think Kinto.js is supposed to keep it :/

Reply:
https://github.com/Kinto/kinto.js/blob/master/src/collection.js#L286